api/licenses/serializers.py

```diff
@@ -11,7 +11,14 @@ class LicenseSerializer(JSONAPISerializer):
         'name',
         'id',
     ])
-    non_anonymized_fields = ['type']
+    anonymized_fields = [
+        'id',
+        'links',
+        'name',
+        'required_fields',
+        'text',
+        'url',
+    ]
```

Comment:
Hardcoding `anonymized_fields` instead of `non_anonymized_fields`. Even though the list is longer here, on the NodeSerializer, this list is going to be much shorter.  Since anonymized_fields are probably an exception instead of the rule, thought we should flip this.

Reply:
Yeah, this makes sense, although I see now why all the confusion. The Anon VOL code was made back when every time we made a change to code, we would expose some anonymous information (maybe a bit of an exaggeration, but not as much of one as I would like). However, our models are much more stable now, so that shouldn't be as big of a deal going forward as it used to be. And users are anonymous by default still.